### ISSUE TITLE
Replace JSON::Tiny by JSON::Fast

### ISSUE THREAD

**Issue by 2colours** [melezhik/sparky#60]
It's meant to be faster and much better maintained, it has a much more integral place within the ecosystem.

The only gotcha I can imagine with this is if there is a test that relies on the formatting of some JSON... other than that, I think it should be a trivial change. I can't do it right now but probably could do it some other time, so making this note.

**Comment by melezhik** [comment]
@2colours Thanks for reporting this, fixed and planned for the next release

**Comment by 2colours** [comment]
Right now there still may be some hidden dependents:

```
 ===SORRY!=== Error while compiling /root/.sparrow6/tmp/216178/1/sparrow6lib.rakumod (sparrow6lib)
02:05:40 :: Could not find JSON::Tiny in:
02:05:40 ::     /root/.sparrow6/tmp/216178/1
02:05:40 ::     /root/.sparrowdo/rad.io-debug/sparrow6/plugins/directory/lib
02:05:40 ::     /root/.sparrowdo/rad.io-debug/sparrow6/plugins/directory/raku-lib
02:05:40 ::     /root/.raku
02:05:40 ::     /opt/rakudo-pkg/share/perl6/site
02:05:40 ::     /opt/rakudo-pkg/share/perl6/vendor
02:05:40 ::     /opt/rakudo-pkg/share/perl6/core
02:05:40 ::     CompUnit::Repository::AbsolutePath<6370684243288>
02:05:40 ::     CompUnit::Repository::NQP<6370639891440>
02:05:40 ::     CompUnit::Repository::Perl5<6370639891480>
02:05:40 :: at /root/.sparrow6/tmp/216178/1/sparrow6lib.rakumod (sparrow6lib):7
02:05:40 ::
02:05:40 :: at /root/.sparrowdo/rad.io-debug/sparrow6/plugins/directory/hook.pl6:1
```

**Comment by melezhik** [comment]
Interesting, can I see a whole scenario code ?

**Comment by melezhik** [comment]
Upss, I found an issue with Sparrow6 itself 

https://github.com/melezhik/Sparrow6/blob/8c198c173ba387787ce70acc6b1b89ef0f4b599c/resources/sparrow6lib.rakumod#L7

**Comment by melezhik** [comment]
@2colours please try to update Sparrow6 from master branch, this should fix the issue, I am going to release a new version of it soon

**Comment by melezhik** [comment]
fixed in Sparrow6 0.0.43 / release to rakuland

**Comment by melezhik** [comment]
released Sparky:ver<0.1.15>:auth<zef:melezhik>
